src/devices/laserHokuyo/laserHokuyo.cpp

```diff
@@ -569,7 +575,6 @@ void laserHokuyo::run()
         // static int countt=0;
         // yDebug() << countt++ << getEstPeriod() << getEstUsed();
     }
-    mutex.unlock();
```

Comment:
I think it is better to leave this and remove the next one, since this is what the code actually did until now...
I'm fixing this.